### PR TITLE
update RSEM; add annotSV

### PR DESCRIPTION
**Pull request by jharenza**
#### Purpose

_Briefly describe your changes here._
- Add Manta annotation, AnnotSV
- Update RSEM methods to clarify both TPM and FPKM were created from RSEM

#### Issue

_What GitHub issue does your pull request address?_
#57 and [manta](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/272)
#### Pull review checklist

The PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)

### CODE REVIEW

File: content/03.methods.md [hunk 1]
```diff
@@ -116,11 +116,12 @@ We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
 Manta SV calling was also limited to regions used in Strelka2.
 The hg38 reference for SV calling used was limited to canonical chromosome regions.
 The somatic DNA workflow for SNV, INDEL, copy number, and SV calling can be found in the [KidsFirst Github repository](https://github.com/kids-first/kf-somatic-workflow).
+Manta SV output was annotated using [AnnotSV](https://lbgi.fr/AnnotSV/) and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).
```
Comment (jaclyn-taroni) [inline content/03.methods.md]: Should include the AnnotSV citation here
```suggestion
Manta SV output was annotated using [AnnotSV](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).
```

Comment (jharenza) [reply content/03.methods.md]: nice catch, i looked quickly at the site and missed that

File: content/03.methods.md [hunk 2]
```diff
@@ -116,11 +116,12 @@ We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
 Manta SV calling was also limited to regions used in Strelka2.
 The hg38 reference for SV calling used was limited to canonical chromosome regions.
 The somatic DNA workflow for SNV, INDEL, copy number, and SV calling can be found in the [KidsFirst Github repository](https://github.com/kids-first/kf-somatic-workflow).
+Manta SV output was annotated using [AnnotSV](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).
```
Comment (jashapiro) [inline content/03.methods.md]: Can we specify the version of AnnotSV that was used?

Comment (jharenza) [reply content/03.methods.md]: @kgaonkar6 - can you alter the CWL to pull the version of AnnotSV used and also let us know which version here? Thank you!

**Comment by kgaonkar6** [comment]
version for annotSV is 2.1 in the dockerfile, I'll update the tag in cwl.

**Pull request merged**